server/services/prison/PrisonRegisterService.spec.ts

```diff
@@ -17,6 +17,16 @@ describe('Magic Link Service', () => {
   })
 
   describe('getActivePrisons', () => {
+    it('should make the request to the prison register without an authorization header', done => {
+      nock(config.apis.prisonRegister.url, {
+        badheaders: ['authorization'],
```

Comment:
Any headers in the request that are in the `badheaders` array will not match the request and will fail the test. This is perfect for what we want to do here 👍